src/util.js

```diff
@@ -55,3 +55,26 @@ exports.extractValueFromObject = (obj, path, def) => {
 
   return current
 }
+
+exports.handleServerError = (ctx, operationFailureMsg, error, logger) => {
+  ctx.status = 500
+  const err = `${operationFailureMsg}${error.message}`
+  ctx.body = {error: err}
+  logger.error(err)
+}
+
+exports.validateEndpoint = body => {
+  const validationError = 'Endpoint validation error: '
+
+  if (!body || !body.name) return `${validationError}name missing`
```

Comment:
@bradsawadye This check seems very repetitive.
This `!body` and `!body.transformation` could be simplified

Reply:
done in https://github.com/jembi/openhim-mediator-mapping/pull/48/commits/b0c7dcd728d8e4fa3da48006e1d24d554ebe65e6